include/FockSpace/EvaluationIterator.hpp

```diff
@@ -168,6 +187,10 @@ class EvaluationIterator<Eigen::SparseMatrix<double>> {
         this->triplet_vector = {};
     }
```

Comment:
Don't spread the documentation over multiple lines.

"Fill the sparse matrix with the elements stored in the triplet vector. The more elements that are already present in the matrix, the more expensive this operation becomes. Therefore, it is ill-advised to call the method more than once.

After filling, the current triplet vector is cleared."